### PR TITLE
Add info on how to use with ember-cli-head to the README.

### PR DESCRIPTION
**Pull request by rwjblue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,3 +117,41 @@ export default Ember.Route.extend({
 This will result in these titles:
 - On /posts - "Posts - My Blog"
 - On /posts/1 - "Ember is Omakase - Posts - My Blog"
+
+### Use with `ember-cli-head`
+
+Using `ember-cli-document-title` with [ember-cli-head](https://github.com/ronco/ember-cli-head)
+is very straight forward and allows you to use the wonderful route based declaritive API for
```
Comment (kimroen) [inline README.md]: Maybe a bit self-praise-y for the README? ^^ I'll take it!

**Comment by kimroen** [comment]
Thanks a bunch. Do you have thoughts on tests to cover this use case?

**Comment by rwjblue** [comment]
The only thing that we need to ensure is tested, is using a custom `Router#setTitle` method.  All of the rest is outside of this packages responsibility (I have only included the info to help future travelers 😝 ).

**Comment by listepo** [comment]
+1

**Comment by kimroen** [comment]
Thank you again - sorry it took so long!

**Pull request merged**